packages/react-magma-dom/src/components/TimePicker/TimePicker.tsx

```diff
@@ -140,7 +143,10 @@ export const TimePicker = React.forwardRef<HTMLInputElement, TimePickerProps>(
             type="number"
             value={hour}
           />
-          <Divider> : </Divider>
+          <Divider isInverse={isInverse} theme={theme}>
```

Comment:
I think these changes snuck out in here, but are already a part of another PR. Should we revert?

Reply:
I do not see this on main.  I think we should keep this in.

Reply:
The changes are in this open PR: https://github.com/cengage/react-magma/pull/605/files

Reply:
Oops, is there a fancy way to just remove the timepicker file?